### ISSUE TITLE
Liquid creates `ErrorBlock` on specific setup

### ISSUE THREAD

**Issue by Kokonico** [Kokonico/Hackulation#1]
Liquid can cause the simulation to crash under a very specific setup

1. place 5 liquid blocks up in the air all on top of each other, like so:

~
~
~
~
~

2. tick 2-3 times. One of the liquid blocks will become an `ErrorBlock`, the total liquid will stay the same however.

**Comment by Kokonico** [comment]
I'm pretty sure I've found how it breaks. it's this code snippet right here (in `classes/block.py`)
```python
if len(NoneBlocks) > 0 and average_liquid < self.surface_tension:
  # we can't flow into the None blocks
  blocks_to_move_to = [block for block in blocks_to_move_to if world.world_array[block[1]][block[0]] is not None]
  # recalculate the average liquid (total liquid is the same)
  average_liquid = total_liquid / (len(blocks_to_move_to) + (1 if self.liquid_level > self.threshold else 0))
```
if the liquid block can only flow to `None` spaces AND it's liquid level is below the threshold and surface tension, it's possible to have no blocks_to_move_to and not add itself, causing it to divide by 0.

**Comment by Kokonico** [comment]
fixed with #2
